src/rm_tdigest.c

```diff
@@ -604,7 +606,7 @@ void *TDigestRdbLoad(RedisModuleIO *rdb, int encver) {
     const double compression = RedisModule_LoadDouble(rdb);
     td_histogram_t *tdigest = td_new(compression);
     tdigest->min = RedisModule_LoadDouble(rdb);
-    tdigest->min = RedisModule_LoadDouble(rdb);
+    tdigest->max = RedisModule_LoadDouble(rdb);
```

Comment:
Do we have any tests to cover RDB?
If not, lets add them in another PR.

Reply:
I believe we don't test enough the rdb load otherwise we should have caught this before. 
I've reverted TDigestRdbLoad fix given its not related to this PR (addressing separately)